src/store/flow/block-types/MobilePrimitives_NumericResponseBlockStore.ts

```diff
@@ -68,21 +69,45 @@ export const actions: ActionTree<IFlowsState, IRootState> = {
         prompt: blankResource.uuid,
         validation_minimum: undefined,
         validation_maximum: undefined,
+        ...await dispatch('initiateExtraVendorConfig'),
       },
       tags: [],
       vendor_metadata: {},
     })
   },
 
+  /**
+   * Override this in the consumer side to add extra config props to avoid the validation saying we have missing prop at the creation
+   * eg: {
+   *   prop1: undefined,
+   *   prop2: undefined,
+   * }
+   */
+  async initiateExtraVendorConfig(_ctx: unknown): Promise<object> {
```

Comment:
Is this function asynchronous in case users need it to be such (right now we don't require it to be)?

Reply:
For some reason, I was only able to override this from voto5 if it's an `async`, I don't know why. To not spend times understanding the working code, I decided to keep it async :D